examples/jose/query.py

```
@@ -0,0 +1,427 @@
+                "working_directory": WORKING_DIR,
+                "model": self.model_name,
+                "conversation_history_length": len(self.conversation_history) // 2,  # Pairs of messages
+            }
+
+            # Try to get more detailed information from the storages
+            if hasattr(self.rag, "text_chunks") and self.rag.text_chunks:
+                # Count documents
+                try:
+                    all_keys = await self.rag.text_chunks.get_all_keys()
+                    info["document_count"] = len(all_keys)
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)